.github/workflows/default.yml

```diff
@@ -99,7 +99,12 @@ jobs:
         uses: usefulness/agp-version-finder-action@v1
 
       - id: build-agp-matrix
-        run: echo '::set-output name=agp-versions::["${{ steps.agp-version-finder.outputs.latest-stable }}", "${{ steps.agp-version-finder.outputs.latest-alpha }}"]'
+        run: |
+           if [[ ${{ github.event_name }} == 'pull_request' ]]; then
+             echo 'agp-versions=["${{ steps.agp-version-finder.outputs.latest-stable }}"]' >> $GITHUB_OUTPUT
+           else
+             echo 'agp-versions=["${{ steps.agp-version-finder.outputs.latest-stable }}", "${{ steps.agp-version-finder.outputs.latest-alpha }}"]' >> $GITHUB_OUTPUT
+           fi
```

Comment:
I considered having a separate workflow for each scenario (build main branch vs build pull requests), but decided it's not worth separating yet

Reply:
Yeah, this seems good enough.